simple_app/utils.py

```diff
@@ -111,7 +134,16 @@ class SearchForm(FlaskForm):
     """
     Searchbar class
     """
-    search = StringField('Search for an object:', id='autocomplete')  # searchbar
+    search = StringField('Search for an object:', id='mainsearchfield')  # searchbar
+    refsearch = StringField('Filter by reference or comment:', id='refsearchfield')
```

Comment:
Do we need the Filter by reference or comment part if we also have the Loose search? The later does the same and is a little more powerful.

Reply:
as discussed